dynawo/sources/Common/DYNTrace.cpp

```diff
@@ -62,8 +62,6 @@ namespace keywords = boost::log::keywords;
 namespace DYN {
 
 static vector< boost::shared_ptr<Trace::TextSink> > originalSinks;
```

Comment:
This one remains static?

Reply:
It is static at file scope, not class scope

Reply:
yes, but it was modified due to similar issues, so why this one is different from the others 2 in this version?

Reply:
reintegrated as a class member: working as well